### PR TITLE
[DOCS-6355] Update uninstall instructions

### PR DESCRIPTION
**Pull request by brett0000FF**
### What does this PR do? What is the motivation?

- Replace "restart your container" with the specific command for Docker in Single Step APM Instrumentation.
- Add new uninstall section to Local Library Injection with same content as Single Step page.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Review by buraizu — APPROVED** [review]
LGTM

**Comment by brett0000FF** [comment]
@Kyle-Verhoog - Could you please help me verify these changes? Vidur already took a look, and he suggested that I combine the tabs on the Local Library Injection page to match the existing ones. For context, this content came from the Single Step APM Instrumentation page.

Do the steps make sense to you, and do you think it makes sense to revise the tabs like so:
Linux host or VM -> Host
Docker -> Agent and app in separate containers

File: content/en/tracing/trace_collection/library_injection_local.md [hunk 1]
```diff
@@ -1020,6 +1018,66 @@ Exercise your application to start generating telemetry data, which you can see
...
+   ```
+2. Restart the service.
+
+{{% /tab %}}
+
+{{% tab "Agent and app in separate containers" %}}
+
+1. Add the `DD_INSTRUMENT_SERVICE_WITH_APM` environment variable to the service startup command: 
+   ```shell
+   docker run -e DD_INSTRUMENT_SERVICE_WITH_APM=false
```
Comment (Kyle-Verhoog) [inline content/en/tracing/trace_collection/library_injection_local.md:1043]: (not sure what the convention is so feel free to disregard)

```suggestion
   docker run -e DD_INSTRUMENT_SERVICE_WITH_APM=false ...
```

**Review by Kyle-Verhoog — APPROVED** [review]
lgtm! Thanks @brett0000FF!

**Comment by brett0000FF** [comment]
@Kyle-Verhoog - Thanks for reviewing! I think I'll leave the `...` off the command because there is a copy button on the example, so I would prefer to leave it as something that can run if copied/pasted.

**Pull request merged**